nomos-services/network/src/backends/libp2p/command.rs

```diff
@@ -21,6 +21,16 @@ pub enum Command {
         message: Box<[u8]>,
         retry_count: usize,
     },
+
+    // Kademlia commands
+    GetClosestPeers {
+        peer_id: PeerId,
+        reply: oneshot::Sender<Vec<PeerInfo>>,
+    },
+
+    DumpRoutingTable {
+        reply: oneshot::Sender<Vec<PeerId>>,
+    },
```

Comment:
I think at this point we should start delegating commands. What I mean is to wrap them into bigger sets:
```rust
pub enum PubSubCommand {
    Broadcast {},
    Subscribre {},
    Unsubscribe {},
    ....
}

pub enum DiscoveryCommand {
    GetClosestPeers {},
    DumpRoutingTable {}
}

pub enum Command {
    PubSub(PubSubCommand),
    Discovery(DiscoveryCommand)
}

```

Reply:
Yes, this plus handle functions can remove the clippy suppress warnings as well.